### ISSUE TITLE
Copay showed incorrect balance

### ISSUE THREAD

**Issue by cephoe** [bitpay/copay#5970]
When sending 0.6 bitcoins, while the transaction in process, my Copay wallet showed a balance of 0 bitcions and it said that a transaction of 8.8 bitcoins was pending confirmation. All of this was incorrect.

After the transaction was done, it reverted the correct amount. This was very distressing in the meantime. Very disappointed with your app. Please advise,

Justin

**Comment by dabura667** [comment]
If you have only one received transaction in your wallet with 9.4 BTC received value, and you spend 0.6 BTC to someone else, all 9.4 BTC must be used in the transaction.

Think of Bitcoins like dollar bills. You received a 9.4 BTC "dollar bill" and in order to send 0.6 BTC you need to send the 9.4 BTC "bill" out to the network, and the network then attributes 0.6 BTC to the other person and gives you 8.8 BTC in "change" just like how if you paid for $3 item with $5 bill, you get $2 bill back as change.

In Copay, by default, it does not let you spend "locked" or "unconfirmed" funds. So your balance (spendable money) went to 0 while the network was confirming your 0.6 BTC send.

This is how the network works. Other wallets might lie to you and hide all of this by saying "oh, well you have 8.8 BTC balance, everything is OK mate!" under the assumption that everything will go well, but Copay does not lie, it says "you can now only spend 0, because you haven't yet received the change."

However, there is no worry of losing money when sending, because even if the network failed to process your transaction, the 9.4 BTC would just return to the previous state and your balance would be 9.4 BTC again. (this is rare, and usually caused by "economy" fee setting.)

**Comment by gabrielbazan7** [comment]
@cephoe if you click on that box, you could have more info about what's happening in your wallet. 

<img width="398" alt="bitpay_-_secure_bitcoin_wallet" src="https://cloud.githubusercontent.com/assets/10999037/25782389/9c8a3cc4-3320-11e7-99f4-0ab0baa8724f.png">

<img width="402" alt="bitpay_-_secure_bitcoin_wallet_and_copay_showed_incorrect_balance_ _issue__5970_ _bitpay_copay" src="https://cloud.githubusercontent.com/assets/10999037/25782395/be95c388-3320-11e7-9ce5-8f9669f02aeb.png">

In this test wallet for example, I send 1 BTC. You could see there the transaction pending to confirmation. On the more info view, you could see the total balance in your wallet, the balance available in your wallet (you could use the total balance if your go to settings -> advanced options -> and turn on "use unconfirmed funds" ) and the pending balance in your wallet that is the subtraction between the total balance in your wallet and the available balance.

**Comment by SatoshiReport** [comment]
This issue happened to me as well - however it never resolved itself and shows a zero balance.  This is on the iphone app under a watch wallet.  Rather concerning until I was able to confirm that the coins are actually still there.  This is 7 days after my transaction still - so there should be plenty of confirmations in the meantime.  I transferred all my coins into my trezor and then I spent some of them.  It created two transactions as it should.  However my change went back to a wallet address that my trezor knows but not one that copay recognizes.
